packages/web-app-files/src/composables/actions/files/useFileActions.ts

```diff
@@ -0,0 +1,318 @@
+import get from 'lodash-es/get'
+import kebabCase from 'lodash-es/kebabCase'
+import { Store } from 'vuex'
+import { ShareStatus } from 'web-client/src/helpers/share'
+import { routeToContextQuery } from 'web-pkg/src/composables/appDefaults'
+import { configurationManager } from 'web-pkg/src/configuration'
+
+import { isLocationSharesActive, isLocationTrashActive } from '../../../router'
+import { computed, unref } from 'vue'
+import { useRouter, useStore } from 'web-pkg/src/composables'
+import { useGettext } from 'vue3-gettext'
+import { Action, ActionOptions, useIsSearchActive } from 'web-pkg/src/composables/actions'
+
+import {
+  useFileActionsAcceptShare,
+  useFileActionsCopy,
+  useFileActionsDeclineShare,
+  useFileActionsDelete,
+  useFileActionsDownloadArchive,
+  useFileActionsDownloadFile,
+  useFileActionsFavorite,
+  useFileActionsMove,
+  useFileActionsNavigate,
+  useFileActionsRename,
+  useFileActionsRestore,
+  useFileActionsShowEditTags
+} from './index'
+
+export const EDITOR_MODE_EDIT = 'edit'
+export const EDITOR_MODE_CREATE = 'create'
+
+export const useFileActions = ({ store }: { store?: Store<any> } = {}) => {
+  store = store || useStore()
+  const router = useRouter()
+  const { $gettext, interpolate: $gettextInterpolate } = useGettext()
+  const isSearchActive = useIsSearchActive()
+
+  const { actions: acceptShareActions } = useFileActionsAcceptShare({ store })
+  const { actions: copyActions } = useFileActionsCopy({ store })
+  const { actions: deleteActions } = useFileActionsDelete({ store })
+  const { actions: declineShareActions } = useFileActionsDeclineShare({ store })
+  const { actions: downloadArchiveActions } = useFileActionsDownloadArchive({ store })
+  const { actions: downloadFileActions } = useFileActionsDownloadFile()
+  const { actions: favoriteActions } = useFileActionsFavorite({ store })
+  const { actions: moveActions } = useFileActionsMove({ store })
+  const { actions: navigateActions } = useFileActionsNavigate({ store })
+  const { actions: renameActions } = useFileActionsRename({ store })
+  const { actions: restoreActions } = useFileActionsRestore({ store })
+  const { actions: showEditTagsActions } = useFileActionsShowEditTags({ store })
+
+  const systemActions = computed((): Action[] => [
```

Comment:
Was also different before, but IMO should be the same order as in the context menu, i.e.:

```js
...unref(downloadArchiveActions),
...unref(downloadFileActions),
...unref(deleteActions),
...unref(moveActions),
...unref(copyActions),
...unref(renameActions),
...unref(showEditTagsActions),
...unref(restoreActions),
...unref(acceptShareActions),
...unref(declineShareActions),
...unref(favoriteActions),
...unref(navigateActions)
```